src/runtime_src/core/tools/common/SubCmd.h

```diff
@@ -65,8 +65,9 @@ class SubCmd {
   void setLongDescription(const std::string &_longDescription) {m_longDescription = _longDescription; };
   void setExampleSyntax(const std::string &_exampleSyntax) {m_exampleSyntax = _exampleSyntax; };
   void printHelp(const boost::program_options::options_description & _optionDescription,
-                 const boost::program_options::options_description & _optionHidden,
-                 bool removeLongOptDashes = false) const;
+                 const boost::program_options::options_description & _optionHidden,
+                 bool removeLongOptDashes = false,
+                 const std::string customHelpSection = "") const;

```

Comment:
```suggestion
                 const std::string& customHelpSection = "") const;

```
Should compile like this!